assets/openshift-apiserver/openshift-apiserver-deployment.yaml

```diff
@@ -119,6 +153,15 @@ spec:
             cpu: {{ .CPU }}{{ end }}{{ if .Memory }}
             memory: {{ .Memory }}{{ end }}{{ end }}{{ end }}
 {{ end }}
+{{- if .KonnectivityEnabled }}
+        env:
+        - name: "HTTP_PROXY"
+          value: "socks5://127.0.0.1:8090"
```

Comment:
Is 8090 suppose to be `konnectivity_server_cluster_port: 8090`?   If so, please use the variable.

Reply:
Good catch. Updated! Thanks!